extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -24,6 +24,10 @@ amp-sticky-ad {
   /* TODO(zhouyx, #3250): discuss on what the max-height should be? */
   max-height: 100px !important;
   margin-bottom: 0 !important;
+  margin-left: 0 !important;
```

Comment:
Left and right shouldn't be necessary, and may actually be required by the user when showing a 320w ad on a larger phone.